gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -591,9 +596,7 @@ private void unscheduleSpec(URI specURI, String specVersion) throws JobException
       try {
           FlowSpec spec = (FlowSpec) this.flowCatalog.get().getSpecs(specURI);
           Properties properties = spec.getConfigAsProperties();
-          _log.info("Scheduler trigger tracing: [flowName: {} flowGroup: {}] - Unscheduled Spec",
-                  properties.getProperty(ConfigurationKeys.JOB_NAME_KEY),
-                  properties.getProperty(ConfigurationKeys.JOB_GROUP_KEY));
+          _log.info(jobSchedulerTracePrefixBuilder(properties) + "Unscheduled Spec");
```

Comment:
I definitely support the reuse between these... but also see the prefix builder using `FLOW_GROUP_KEY`, while the prior revision had used `JOB_GROUP_KEY`.  can the former stand in for the latter... or was it incorrect previously to use `JOB_GROUP_KEY`?

Reply:
Previously was incorrectly using `JOB_GROUP_KEY`, we should track at `FLOW` level